Formula/q.rb

```diff
@@ -17,14 +15,27 @@ class Q < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "bed14a331133ff96b85fa37e0729ca695bd273f78ee82e792185d137edf9917a"
   end
 
+  depends_on "pyoxidizer" => :build
+  depends_on "python@3.8" => :build
   depends_on "ronn" => :build
-  depends_on "python@3.9"
-  depends_on "six"
+  depends_on xcode: ["12.4", :build]
+
+  # Patch to allow proper ARM builds by using python3.9.
```

Comment:
3.9? The dependency is on Python3.8 though.

Reply:
The python 3.8 is a test dependency (tests are running using it), unrelated to the python 3.9 inside pyoxidizer. Good catch, I'll move it to be a build dependency, or even check that it can be removed altogether.

The parameter of python 3.9 below is something else - It's the version of CPython that pyoxidizer uses internally in order to link and generate the standalone binary, it's a configuration unrelated to the test dependency, and for ARM compatibility it has to be python 3.9, since this is the only version supported right now by pyoxidizer for building on ARM.